picobot/painter.py

```diff
@@ -205,7 +205,13 @@ def sticker_from_text(
 
 
 def wrapped_text(text: str, line_limit=25):
-    words = text.split(' ')
+    splitted = text.split(' ')
+    words = []
+    for word in splitted:
+        words.extend(
+            [word[i : i + line_limit] for i in range(0, len(word), line_limit)]
+        )
+
     lines = [words[0]]
     for w in words[1:]:
         if len(lines[-1]) + len(w) < line_limit:
```

Comment:
Doesn't [`textwrap.wrap`](https://docs.python.org/3/library/textwrap.html#textwrap.wrap) already solve this? As in:

```python
def wrapped_text(text: str, line_limit=25):
    return textwrap.wrap(text, width=line_limit)
```

Reply:
:man_facepalming:

Reply:
I cry evetiem